README.md

```diff
@@ -1,126 +1,188 @@
 # partner-charts
+This repository is reserved for partner charts in the Rancher's v2.5+ catalog.
 
-This is an experimental project to automate system charts and adding overlays files on top of it.
+As part of this catalog, all charts will benefit of a cloud native packaging system
```

Comment:
Are we forcing helm3 on partner charts now ?

Reply:
Yes, because 2.5+ uses Helm 3 only (as far as I know) and right now if they submit a Helm 2 chart we would have to worry about whether is Helm 3 compatible before migrating it to the new catalog.